integration-tests/integration/prisonerProfile/prisonerCellHistory.spec.js

```diff
@@ -27,7 +27,7 @@ context('Prisoner cell history', () => {
             {
               agencyId: 'MDI',
               assignmentDate: '2020-05-01',
-              assignmentDateTime: '2020-05-01T12:48:33.375Z',
+              assignmentDateTime: '2020-05-01T12:48:33.375',
```

Comment:
Defining the dates times like this makes them UTC which will work on our builds but not locally.